templates/projects/tasks_browse.webpack.ejs

```diff
@@ -625,6 +625,33 @@
 
 <script src="<%= htmlWebpackPlugin.files.chunks.task_browse.entry %>"></script>
 <script type="text/javascript">
+    function autoRefresh() {
+      window.location = window.location.href;
+    }
+
+    const proj = {{ project |tojson }};
+    const worker = {{ for_worker |tojson }};
+    const enableAutoRefresh = (proj['enable_task_queue'] && worker);
+    const refreshInterval = 10000;
+    let refresh = null;
+
+    if (enableAutoRefresh) {
+      refresh = setInterval('autoRefresh()', refreshInterval);
+    }
+
+    $(window).focus(function() {
+      if (enableAutoRefresh && !refresh){
+        refresh = setInterval('autoRefresh()', refreshInterval);
+      }
+    });
+
+    $(window).blur(function() {
+      if (refresh) {
+        clearInterval(refresh);
+        refresh = 0;
+      }
+    });
```

Comment:
is it possible to avoid set/clear interval replacing this logic w/ setTimeout? If so, we can also remove autoRefresh definition.
```javascript
if (enableAutoRefresh) {
     window.setTimeout(function () {
          window.location.reload();
     }, refreshInterval);
}
```

Reply:
The purpose with this is when user switches to another application, it would stop auto refresh, when user comes back, it start auto refresh again

Reply:
make sense